docs/developer-guide.md

```diff
@@ -8,7 +8,7 @@ These release branches follow a specific naming format: YYYYx, where "YYYY" repr
 The structure of the notebook's build chain is derived from the parent image. To better comprehend this concept, refer to the following graph.
 
 <p align="center">
-<img src="https://github.com/opendatahub-io/notebooks/assets/42587738/9e5df03f-01f4-4bba-9792-3b56e9b5b912" data-canonical-src="https://github.com/opendatahub-io/notebooks/assets/42587738/9e5df03f-01f4-4bba-9792-3b56e9b5b912" width="820" height="520" />
+<img src="https://github.com/opendatahub-io/notebooks/assets/42587738/9e5df03f-01f4-4bba-9792-3b56e9b5b912" data-canonical-src="https://github.com/opendatahub-io/notebooks/assets/42587738/9e5df03f-01f4-4bba-9792-3b56e9b5b912" width="820" height="450" />
```

Comment:
I'm afraid that it may end up being as extensive as it appears in the readme, where you barely see information about the notebook's origins, etc. In this section, it would be nice to present the concept of a top-down structure. I guess I have to clarify it in this section.

Reply:
Okay, agreed :slightly_smiling_face: